test/passing/fun_decl.ml

```diff
@@ -1,77 +1,32 @@
 [@@@ocamlformat "wrap-fun-args=false"]
 
-class t =
-  object
-    method meth
-          aaaaaaaaaaa
-          bbbbbbbbbbbbbb
-          ccccccccccccccccccc
-          ddddddddddddddddddddd
-          eeeeeeeeeeeeeee =
-      body
-  end
-
-let func
-    aaaaaaaaaaa
-    bbbbbbbbbbbbbb
-    ccccccccccccccccccc
-    ddddddddddddddddddddd
-    eeeeeeeeeeeeeee =
-  body
-
-let rec func
-    aaaaaaaaaaa
-    bbbbbbbbbbbbbb
-    ccccccccccccccccccc
-    ddddddddddddddddddddd
-    eeeeeeeeeeeeeee =
-  body
-
 let to_loc_trace
     ?(desc_of_source =
       fun source ->
         let callsite = Source.call_site source in
         Format.asprintf
           "return from %a"
           Typ.Procname.pp
-          (CallSite.pname callsite))
-    ?(source_should_nest = fun _ -> true)
+          (CallSite.pname callsite)) ?(source_should_nest = fun _ -> true)
```

Comment:
This regression should be fixed, `source_should_nest` should be indented on the same level as `desc_of_source` and the other function's parameters.